fnl/modules/tools/debugger/config.fnl

```diff
@@ -24,11 +24,27 @@
                     (vim.fn.input "Path to executable: "
                                   (.. (vim.fn.getcwd) "/") :file))}])
 
+(set dap.adapters.coreclr
+     {:type :executable
+     :command "/usr/local/bin/netcoredbg/netcoredbg"
+     :args ["--interpreter=vscode"]})
+
+(local coreclr-configs
+       [{:name "netcoredbg"
+        :type :coreclr
+        :request :launch
+        :program (fn []
+                   (vim.fn.input "Path to executable: "
+                                 (.. (vim.fn.getcwd) "/bin/Debug/" :file)))
+       }])
+
 (nyoom-module-p! cc
                  (do
                    (set dap.configurations.c lldb-configs)
                    (set dap.configurations.c dap.configurations.cpp)))
 
+(nyoom-module-p! csharp (set dap.configurations.cs coreclr-configs))
+
```

Comment:
should this not be moved to a `(do [...])` block inside the `nyoom-module-p!` call for the `csharp` module?

```suggestion
(local coreclr-configs
       [{:name "netcoredbg"
        :type :coreclr
        :request :launch
        :program (fn []
                   (vim.fn.input "Path to executable: "
                                 (.. (vim.fn.getcwd) "/bin/Debug/" :file)))
       }])

(nyoom-module-p! cc
                 (do
                   (set dap.configurations.c lldb-configs)
                   (set dap.configurations.c dap.configurations.cpp)))

(nyoom-module-p! csharp (do
                                                   (set dap.configurations.cs coreclr-configs)
                                                   (set dap.adapters.coreclr
                                                        {:type :executable
                                                        :command "/usr/local/bin/netcoredbg/netcoredbg"
                                                        :args ["--interpreter=vscode"]})))
```

Reply:
For now I'd prefer to keep all of the debugger stuff in the debugger module, it's what doom does. Not sure if I'll keep it in one file in the future, but for now its easier to work with (and keeps merge conflicts to a minimum)

Reply:
Oh my mistake, I read it incorrectly. Indeed it should be moved to a `do` block

Reply:
Great, I'll update the PR today. Thanks for the feedback!

Reply:
Pushed an update @hbjydev, do you want me to resolve this or should it be you? Not sure if there's a policy around that or not is all.

Reply:
No policy, looks good to me. thanks!